net/tstun/wrap.go

```diff
@@ -64,7 +65,8 @@ type Wrapper struct {
 
 	closeOnce sync.Once
 
-	lastActivityAtomic int64 // unix seconds of last send or receive
+	idleDuration syncs.AtomicInt64 // approximate number of seconds since last activity
```

Comment:
call it `idleSeconds` instead of duration? a duration is also an int64 and suggests nanoseconds kinda.